staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -121,10 +123,15 @@ type Event struct {
 	// +optional
 	RequestObject *runtime.Unknown
 	// API object returned in the response, in JSON. The ResponseObject is recorded after conversion
-	// to the external type, and serialized as JSON.  Omitted for non-resource requests.  Only logged
+	// to the external type, and serialized as JSON. Omitted for non-resource requests.  Only logged
 	// at Response Level.
 	// +optional
 	ResponseObject *runtime.Unknown
+
+	// Time the request reached the apiserver.
+	RequestReceivedTimestamp metav1.MicroTime
+	// Time the request reached each audit stage.
+	StageTimestamp metav1.MicroTime
```

Comment:
for new fields we don't need v1beta**2**. Are these the only changes?

Reply:
Yes, they are.
Will update it.

Reply:
Done.
This will break unit test until https://github.com/kubernetes/kubernetes/pull/53050 get mreged.